tests/test_equil.py

```diff
@@ -357,6 +358,40 @@ def test_ntequl():
     pd.testing.assert_frame_equal(df, df_from_inc, check_dtype=False)
 
 
+@pytest.mark.parametrize(
+    "somefloat, expected",
+    [
+        (1000.0000000000003, " 1000.0 "),  # All these decimals would crash Eclipse
+        (1000.0000003, " 1000.0 "),
+        (1000.000003, " 1000.000003 "),  # Eclipse should accept this.
+        (np.float32(1000.00003), " 1000.0 "),
+        (np.float32(1000.0003), " 1000.0003"),  # can give 1000.000305
+    ],
+)
+def test_eclipse_rounding(somefloat, expected):
+    """Values in include files with a lot of decimals, like you sometimes get
```

Comment:
"a lot of decimals" --> replace with exact number of decimals/digits/resolution if known?

Reply:
(I don't know it.. will need testing in Eclipse)